Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertFloor.cs

```diff
@@ -240,17 +240,16 @@ private ICurve GetFlattenedCurve(ICurve curve, double z)
 
         case OG.Polyline poly:
           var polylinePonts = new List<double>();
-          var originalPolylinePoints = poly.ToList();
+          var originalPolylinePoints = poly.points;
           var polyConversionFactor = z * Speckle.Core.Kits.Units.GetConversionFactor(ModelUnits, poly.units);
```

Comment:
I realize that you didn't write this (I did) but this line is wrong. We shouldn't be multiplying this value by z. It should just be ```var polyConversionFactor = Speckle.Core.Kits.Units.GetConversionFactor(ModelUnits, poly.units);```